src/main/resources/templates/BaseEmail.ftl

```diff
@@ -3,9 +3,25 @@
     emails that use this must implement a macro called `EmailBody` and the whole html email will
     eventually get rendered by the statement <@HtmlEmail/>.
 -->
+<style>
```

Comment:
Interesting. It looks like the `<style>` tag got inserted within a sparkpost-generated `<head>` tag.

Reply:
Oh interesting, should I wrap the styling in a <head> tag here? Or is it fine to let Sparkpost handle that?

Reply:
I think it should be fine.